common/include/PS2Eext.h

```diff
@@ -216,44 +215,8 @@ static void SysMessage(const char *fmt, ...)
     if (msg[strlen(msg) - 1] == '\n')
         msg[strlen(msg) - 1] = 0;
 
-    GtkWidget *dialog;
-    dialog = gtk_message_dialog_new(NULL,
-                                    GTK_DIALOG_DESTROY_WITH_PARENT,
-                                    GTK_MESSAGE_INFO,
-                                    GTK_BUTTONS_OK,
-                                    "%s", msg);
-    gtk_dialog_run(GTK_DIALOG(dialog));
-    gtk_widget_destroy(dialog);
-}
-
-static void __forceinline set_logging(GtkToggleButton *check, int &log)
-{
-    log = gtk_toggle_button_get_active(check);
-}
-
-static void __forceinline PluginNullConfigure(std::string desc, int &log)
-{
-    GtkWidget *dialog, *label, *check_box;
-
-    /* Create the widgets */
-    dialog = gtk_dialog_new();
-    label = gtk_label_new(desc.c_str());
-    check_box = gtk_check_button_new_with_label("Logging");
-
-    gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(check_box), (log != 0));
-
-    /* Ensure that the dialog box is destroyed when the user clicks ok, and that we get the check box value. */
-    g_signal_connect(check_box, "toggled", G_CALLBACK(set_logging), &log);
-
-    /* Add all our widgets, and show everything we've added to the dialog. */
-    gtk_container_add(GTK_CONTAINER(gtk_dialog_get_content_area(GTK_DIALOG(dialog))), label);
-    gtk_container_add(GTK_CONTAINER(gtk_dialog_get_content_area(GTK_DIALOG(dialog))), check_box);
-    gtk_dialog_add_button(GTK_DIALOG(dialog), "Ok", 0);
-
-    gtk_widget_show_all(dialog);
-
-    gtk_dialog_run(GTK_DIALOG(dialog));
-    gtk_widget_destroy(dialog);
+    wxMessageDialog *dial = new wxMessageDialog(NULL, msg, wxT("Info"), wxOK);
```

Comment:
This doesn't get deleted, so I think it'll cause a memleak.

I don't think it's necessary to use a pointer in this case since it isn't a child window. You should be able to just do
```
wxMessageDialog dialog(nullptr, msg, "Info", wxOK);
dialog.ShowModal();
```

Reply:
Alright, taken care of.